lte/gateway/configs/templates/mme.conf.template

```diff
@@ -352,4 +353,8 @@ NGAP :
     # DNS address communicated to UEs
     DEFAULT_DNS_IPV4_ADDRESS     = "{{ ipv4_dns }}";
     DEFAULT_DNS_SEC_IPV4_ADDRESS = "{{ ipv4_sec_dns }}";
+    AMF_NAME = "{{ amf_name }}";
+    PLMN_SUPPORT_LIST = (
+       { MCC="{{ mcc }}" ; MNC="{{ mnc }}"; DEFAULT_SLICE_SERVICE_TYPE="{{ default_slice_service_type }}" ;                         DEFAULT_SLICE_DIFFERENTIATOR="{{ default_slice_differentiator }}"; }
```

Comment:
whys is there so many blank spaces before `DEFAULT_SLICE_DIFFERENTIATOR`?

Reply:
Comments addressed.